File: Level1/13.문자열다루기기본/13.문자열다루기기본.cpp
#include <string>
#include <vector>

using namespace std;

bool solution(string s) {
    bool answer = true;
    if(s.size()==4 || s.size()==6){
        answer = true;
    }else{
        return false;
    }
    for(int i = 0 ; i<s.size() ; i++){
        if(s[i]<'0' || s[i]>'9'){
            answer = false;
            break;
        }
    }
    return answer;
}

File: Level1/13.문자열다루기기본/13.문자열다루기기본.py
def solution(s):
    answer = True
    if(len(s)==4 or len(s)==6):
        answer = True
    else:
        return False
    
    for i in range(len(s)):
        if s[i]<'0' or s[i]>'9':
            answer = False
            break
    return answer